content/docs/about/policies/global-attachment.md

```diff
@@ -0,0 +1,380 @@
+
+   ```yaml
+   kubectl apply -f- <<EOF  
+   apiVersion: {{< reuse "docs/snippets/trafficpolicy-apiversion.md" >}}
+   kind: {{< reuse "docs/snippets/trafficpolicy.md" >}}
+   metadata:
+     name: transformation
+     namespace: httpbin
+   spec:
+     targetRefs:
+     - group: gateway.networking.k8s.io
+       kind: HTTPRoute
+       name: httpbin
+     transformation:
+       request:
+         set:
+         - name: ":path"
+           value: '{% if request_header("transform") == "status" %}/status/418{% else %}{{ header(":path") }}{% endif %}'
+   EOF
+   ```
+
+3. Repeat the request with the `transform:status` header. The request is now transformed from the `/anything` path to the `/status/418` path.
+
+   {{< tabs items="Cloud Provider LoadBalancer,Port-forward for local testing" tabTotal="2" >}}
+   {{% tab tabName="Cloud Provider LoadBalancer" %}}
+   ```sh
+   curl -i http://$INGRESS_GW_ADDRESS:8080/anything -H "host: www.example.com:8080" -H "transform: status"
+   ```
+   {{% /tab %}}
+   {{% tab tabName="Port-forward for local testing"%}}
+   ```sh
+   curl -i localhost:8080/anything -H "host: www.example.com" -H "transform: status"
+   ```
+   {{% /tab %}}
+   {{< /tabs >}}
+
+   Example output: Notice that the response is now `418 I'm a teapot!` instead of the initial 200 response.
+   
+   ```txt
+   HTTP/1.1 418 Unknown
+   access-control-allow-credentials: true
+   access-control-allow-origin: *
+   x-more-info: http://tools.ietf.org/html/rfc2324
+   date: Wed, 30 Jul 2025 15:45:07 GMT
+   content-length: 13
+   content-type: text/plain; charset=utf-8
+   x-envoy-upstream-service-time: 1
+   server: envoy
+   
+   I'm a teapot!
+   ```
+
+4. Delete the {{< reuse "docs/snippets/trafficpolicy.md" >}}.
+
+   ```sh
+   kubectl delete {{< reuse "docs/snippets/trafficpolicy.md" >}} -n httpbin transformation
+   ```
+
+5. Create the same {{< reuse "docs/snippets/trafficpolicy.md" >}} but in a different namespace, such as the `{{< reuse "docs/snippets/namespace.md" >}}` namespace.
+
+   ```yaml
+   kubectl apply -f- <<EOF  
+   apiVersion: {{< reuse "docs/snippets/trafficpolicy-apiversion.md" >}}
+   kind: {{< reuse "docs/snippets/trafficpolicy.md" >}}
+   metadata:
+     name: transformation
+     namespace: {{< reuse "docs/snippets/namespace.md" >}}
+   spec:
+     targetRefs:
+     - group: gateway.networking.k8s.io
+       kind: HTTPRoute
+       name: httpbin
+     transformation:
+       request:
+         set:
+         - name: ":path"
+           value: '{% if request_header("transform") == "status" %}/status/418{% else %}{{ header(":path") }}{% endif %}'
+   EOF
+   ```
+
+6. Repeat the request. This time, the request is not transformed because the policy cannot be attached since the resource is in a different namespace. Instead, you get the initial 200 response.
```

Comment:
```suggestion
6. Repeat the request. This time, the request is not transformed. The policy cannot be attached because the resource is in a different namespace. Instead, you get the initial 200 response.
```